tests/protocol/pool-rwa/Pool.Emode.Rwa.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {PoolEModeTests} from 'tests/protocol/pool/Pool.Emode.sol';
+
+contract PoolEModeRwaTests is PoolEModeTests {
+  function setUp() public override {
+    super.setUp();
+    _upgradeToRwaAToken(tokenList.usdx, 'aUsdx');
+  }
+
+  function test_liquidations_shouldApplyEModeLBForEmodeAssets(uint256 amount) public override {
+    vm.prank(poolAdmin);
+    contracts.poolConfiguratorProxy.setLiquidationProtocolFee(tokenList.usdx, 0);
+
+    super.test_liquidations_shouldApplyEModeLBForEmodeAssets(amount);
+  }
+
+  function test_setUserEMode_shouldAllowSwitchingIfNoBorrows(uint8 eMode) public override {
```

Comment:
can you flip the order of these functions? so they follow the same order as the canonical test

Reply:
[68e7e00](https://github.com/aave/aave-v3-horizon/pull/8/commits/68e7e00fdbc44e790d81958c1446ba915cdaaec3)